coding-concepts.js

```diff
@@ -7,48 +7,48 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+//console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ["tangerine", "magenta", "lilac", "daffodil", "indigo"]
+// b) Verify and explain: partially correct. adds the new item to the end of the array, but it returns the new length as a number. there are now 5 strings in the array instead of 4, so the new length is 5.
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+//console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: length method will count the number of characters in a string, including the spaces
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "o"
+// b) Verify and explain: the index will count the characters inside the string. since its an index, it will start the count at zero.
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
-// console.log(languages[index])
+//const index = 1
+//console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: index will count each word within the array of the const declaration named "languages". Index starts the count at zero, but a variable was made to specify an index of 1. "Javascript" would be index = 0.
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+//console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will change all the characters from lowercase to uppercase for both words
+// b) Verify and explain: I was incorrect, as it came out as a "TypeError" in the terminal. weekendDays.toUpperCase is not a function. 
```

Comment:
You should be explaining why as well. The `.toUpperCase()` method only works on strings. In this example, it is being used on an array, which is why it resulted in a TypeError.